pegasus/src/forms.rb

```diff
@@ -1,34 +1,8 @@
-require 'cdo/poste'
 require 'cdo/regexp'
 require src_dir 'database'
+require File.join(CDO.root_dir, 'lib/forms/pegasus_form_errors')
```

Comment:
Since its outside pegasus, I figured this absolute path from the root dir would be cleaner, but I'd be happy to change it to `require_relative '../../lib/forms/pegasus_form_errors'` if you prefer.